script.js

```diff
@@ -0,0 +1,54 @@
+class Numberdrome {
+  constructor() {
+    this.listOfNumbers = [];
+  }
+
+  addNumber(n) {
+    this.listOfNumbers.push(n);
+  }
+
+  removeNumber(n) {
+    while(this.listOfNumbers.includes(n)) {
+      this.listOfNumbers.splice(this.listOfNumbers.indexOf(n), 1);
+    }
+    return this.listOfNumbers;
+  }
+
+  sum() {
+    if (this.listOfNumbers.length === 0){
+      return 0;
+    }
+    return this.listOfNumbers.reduce((sum, current) => sum + current);
+  }
+
+  product() {
+    if (this.listOfNumbers.length === 0){
+      return 1;
+    }
+    return this.listOfNumbers.reduce((sum, current) => sum * current);
```

Comment:
use initialValue in `reduce` instead of `if construction`
https://developer.mozilla.org/uk/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce